packages/paste-website/src/assets/css/fonts.css

```diff
@@ -0,0 +1,424 @@
+  font-weight: 700;
+  font-style: normal;
+}
+
+/**
+   * Colfax font family
+   */
+@font-face {
+  font-family: Colfax;
+  font-style: normal;
+  font-weight: 300;
+  src: url(/fonts/colfax-light.woff2) format('woff2'), url(/fonts/colfax-light.woff) format('woff');
+}
+
+@font-face {
+  font-family: Colfax;
+  font-style: normal;
+  font-weight: 400;
+  src: url(/fonts/colfax-regular.woff2) format('woff2'), url(/fonts/colfax-regular.woff) format('woff');
+}
+
+@font-face {
+  font-family: Colfax;
+  font-style: normal;
+  font-weight: 600;
+  src: url(/fonts/colfax-medium.woff2) format('woff2'), url(/fonts/colfax-medium.woff) format('woff');
+}
+
+/**
+ * Inter font family
+ */
+
+/* cyrillic-ext */
+@font-face {
```

Comment:
Should we also add these to Storybook? https://github.com/twilio-labs/paste/blob/a88611afc794b612cd55770fd8607ee75557757c/.storybook/addons/theme-switcher-addon/index.js

Reply:
Done